jerry-ext/include/jerryscript-ext/extfunc.h

```diff
@@ -0,0 +1,57 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef JERRYX_EXTFUNC_H
+#define JERRYX_EXTFUNC_H
```

Comment:
This is just source tree organization. It will/should not be reflected in the include directives. Basically, "jerry-ext/include" is on the include path and users will still have to use `#include "jerryscript-ext/extfunc.h"` (or, as another example, `#include "jerryscript-ext/args.h"` for the argument validator) only in the sources. And having that "include" path component under "jerry-ext" serves two causes: 1) eases header installation with cmake, and 2) consistency (all our libraries follow that sheme -- except for "jerry-core" but I'm trying to fix that up in #1793).